docs/stressors.md

```diff
@@ -109,9 +111,11 @@ func (m *MyStressorTask) ExecTask(endpoint *model.Endpoint, responses *MutexTask
     stressParams := endpoint.MyStressorComplexity
     time.Sleep(stressParams.MyVariable * time.Second)
 
-    ConcatenateMyStressorResponses(responses, &model.MyTaskResponse{
-        Services: []string{fmt.Sprintf("%s/%s", util.ServiceName, endpoint.Name)},
-        Statuses: []string{fmt.Sprintf("sleep: %d", stressParams.MyVariable)},
+    svc := fmt.Sprintf("%s/%s", util.ServiceName, endpoint.Name)
+    ConcatenateMyStressorResponses(responses, &generated.MyTaskResponse{
+        Services: map[string]int{
+            svc: stressParams.MyVariable,
```

Comment:
How is the status of the call  concatenated now? Isn't good to somehow keep it?

Reply:
It's concatenated here: https://github.com/EricssonResearch/cloud-native-app-simulator/pull/75/files#diff-916c110e9f3b568e695ea13a43288c2d4e1a9dbf9b0d1d5dd156b47add1bd56aR148

I changed it to a map because I thought it made it easier to read (in particular the network task, since before it had three arrays of different length). I can change it back if we want to keep the old format.

Reply:
you are right, it's better as a map